### PR TITLE
Laboratory work #4, Irina Pereyaslavtseva - 22FPL1

### PR DESCRIPTION
**Pull request by iriper**

### CODE REVIEW

File: lab_4_fill_words_by_ngrams/target_score.txt [hunk 1]
```diff
@@ -1 +1 @@
-0
+6
```
Comment (artyomtugaryov) [inline lab_4_fill_words_by_ngrams/target_score.txt:1]: what about pipeline? Is it green?

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]

File: lab_4_fill_words_by_ngrams/target_score.txt (hunk 1 above)
Comment (artyomtugaryov) [inline lab_4_fill_words_by_ngrams/target_score.txt:1]: It is not enought

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]

File: lab_4_fill_words_by_ngrams/target_score.txt (hunk 1 above)
Comment (artyomtugaryov) [inline lab_4_fill_words_by_ngrams/target_score.txt:1]: It is not enough

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]

**Comment by vitazaytseva** [comment]
Well done! Good job!

**Pull request closed without merging**